.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - 3.4
   - 3.5
   - 3.6
+  - 3.7
```

Comment:
Can we add Python 3.8 as well? https://docs.travis-ci.com/user/languages/python/#specifying-python-versions